js/mobile-menu.js

```diff
@@ -15,4 +16,28 @@ function hideMobileMenu() {
 
 hamburger.addEventListener('click', displayMobileMenu);
 xButton.addEventListener('click', hideMobileMenu);
-mobileNav.addEventListener('click', hideMobileMenu);
\ No newline at end of file
+mobileNav.addEventListener('click', hideMobileMenu);
+
+// modal for project details
+const projectsDetails = document.querySelectorAll('.see-project');
+const modal = document.querySelector('.modal-project');
+const xModalButton = document.querySelector('.x-cross-modal');
+const detailsButtons = document.querySelectorAll('.modal-buttons-p');
+
+function displayProjectDetails() {
+  modal.style.display = 'block';
+}
+
+function closeProjectDetails() {
+  modal.style.display = 'none';
+}
+
+projectsDetails.forEach((project) => {
+  project.addEventListener('click', displayProjectDetails);
+});
+
+xModalButton.addEventListener('click', closeProjectDetails);
+
+detailsButtons.forEach((button) => {
+  button.addEventListener('click', closeProjectDetails);
```

Comment:
As mentioned by the previous reviewer, the project popup should be dynamically added to the DOM. This means you will have an array of projects like this 👇🏾 

```javascript
const projects = [{
    name: "Submarine game",
    description: "A Phaser3 platform game built as part of the Microverse JavaScript program.",
    image: "link_to_image.jpg",
    technologies: ["Phaser3", "JavaScript", "Jest"],
    link: "link_to_live_version.com",
}] // You will have about six projects in this array.
```

- Thereafter, the projects section is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). **This requires changes in existing code - already created HTML code for the projects list should be removed, and instead it should be added via JavaScript.** Check an example of student's projects: https://github.com/olawale-o/portfolio-mobile-version/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95 and https://github.com/nicupop729/Portfolio/pull/11/commits/aabed37cc2208f52d0bd081a4cae8cdc83c49636.

- Modal popup data is populated with the data from the projects array.
- Thanks to the previous 2 points, adding projects in the future will be easy. You will only add it in one place - as another object in the projects array.

Reply:
finish modal pop up data is populated from json data